app/controllers/application_controller.rb

```diff
@@ -5,7 +5,7 @@ class ApplicationController < ActionController::Base
 
   rescue_from ActiveRecord::RecordNotFound, with: :not_found
 
-  before_action :check_staging_auth, except: :check
+  # before_action :check_staging_auth, except: :check
```

Comment:
We disable staging HTTP basic auth on staging for now

Reply:
:+1: Let's not leave it in as comment lint for too long, but I'm ok with it while we're still polishing things.